packages/server/graphql/types/TeamDrawerEnum.ts

```diff
@@ -0,0 +1,12 @@
+import {GraphQLEnumType} from 'graphql'
+
+const TeamDrawer = new GraphQLEnumType({
```

Comment:
this is a great single source of truth!
it might also be nice to export a string union here so you can use it on the server: `export type TeamDrawer = 'agenda' | 'manageTeam'